openstack/cronus/ci/test-values.yaml

```diff
@@ -8,3 +8,9 @@ rhea:
     name: rhea-repository
     pullPolicy: IfNotPresent
     tag: latest
+
+maillog:
+  image:
+    name: maillog-repository
+    pullPolicy: IfNotPresent
+    tag: latest
```

Comment:
maillog is not enabled here meaning that you don't get the CI pipeline error checking as your maillog resources won't be included